src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -50,10 +50,15 @@ def acr_import(cmd,
     if source_registry:
         if is_valid_resource_id(source_registry):
             source = ImportSource(resource_id=source_registry, source_image=source_image)
-
         else:
             registry = get_registry_from_name_or_login_server(cmd.cli_ctx, source_registry, source_registry)
             if registry:
+                # trim away redundant login server name, a common error
+                prefix = registry.login_server + '/'
```

Comment:
how about add validation earlier at params?

Reply:
This change will be a bit involving, as to do it right, other 3 parameters should also be moved to the validator not just this one. Can I log an issue and do it later?

Reply:
Logged https://github.com/Azure/azure-cli/issues/13415